setup.py

```diff
@@ -238,9 +238,8 @@ def check_file_at_path(path2file):
     else:
         print("TBB not found")
 
-    # Disable OpenMP if we are building a wheel or
-    # forced by user with NUMBA_NO_OPENMP=1
-    if is_building_wheel() or os.getenv('NUMBA_NO_OPENMP'):
```

Comment:
is `is_building_wheel` now dead?

Reply:
No, it is still used on line 29 in that file.

Reply:
Are you sure? https://github.com/numba/numba/pull/5944/files#diff-2eeaed663bd0d25b7e608891384b7298R29

Reply:
Oh,my bad, that is the definition...